Formula/mcap.rb

```diff
@@ -0,0 +1,47 @@
+class Mcap < Formula
+  desc "Serialization-agnostic container file format for pub/sub messages"
+  homepage "https://mcap.dev"
+  url "https://github.com/foxglove/mcap/archive/refs/tags/releases/mcap-cli/v0.0.6.tar.gz"
+  sha256 "f2adc1a2c1f81d4c5f5b684829013dcdf28f24f15ce34e4a72139ee01355d93a"
+  license "Apache-2.0"
+  head "https://github.com/foxglove/mcap.git", branch: "main"
+
+  depends_on "go" => :build
+
+  resource "testdata" do
```

Comment:
```suggestion
  resource "homebrew-testdata" do
```
This allows the update script to ignore it